tests/lib/external/snapd-testing-tools/utils/log_helper.py

```diff
@@ -0,0 +1,194 @@
+"""
+This library provides a set of functions that can be used to
+process the spread output.
+"""
+
+import re
+
+from enum import Enum
+
+
+class ListedEnum(Enum):
+    @classmethod
+    def list(cls):
+        return list(map(lambda c: c.value, cls))
```

Comment:
This should return `cls.__members__.values()`.

Add a type annotation please.

Reply:
I think this only returns the mappings, so you'd have to do `[v.value for v in cls.__members__.values()]` to get the list of actual string values associated with each member